xml/mysql_banners.xml

```diff
@@ -1035,14 +1192,103 @@
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Ubuntu"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.version" value="20.04"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:20.04"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4})-MariaDB-mariadb\d\Smaverick$" flags="REG_ICASE">
+    <description>MariaDB MariaDB running on a Ubuntu Maverick (10.10)</description>
+    <example service.version="5.5.29">5.5.29-MariaDB-mariadb1~maverick</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="Ubuntu"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.version" value="10.10"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:canonical:ubuntu_linux:10.10"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4}(?:\-\d)?)-MariaDB-alt(\d{1,2})(?:-log)?.*$" flags="REG_ICASE">
+    <description>MariaDB MariaDB running on a ALT Linux</description>
+    <example service.version="10.1.29" os.version="1">5.5.5-10.1.29-MariaDB-alt1.M80P.1</example>
+    <example service.version="10.2.15" os.version="2">5.5.5-10.2.15-MariaDB-alt2.M80P.3.S1</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:mariadb:mariadb:{service.version}"/>
+    <param pos="0" name="os.vendor" value="ALT"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="2" name="os.version"/>
+    <param pos="0" name="os.cpe23" value="cpe:2.3:o:altlinux:alt_linux:{os.version}"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\d{1,2}\.\d{1,3}\.[a-f\d]{1,3}-)?(\d{1,2}\.\d{1,3}\.[a-f\d]{1,4}\-\d{1,2})-MariaDB-enterprise(?:-log)?$" flags="REG_ICASE">
+    <description>MariaDB Maria DB Enterprise Edition</description>
+    <example service.version="10.4.10-4">5.5.5-10.4.10-4-MariaDB-enterprise-log</example>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.vendor" value="MariaDB"/>
+    <param pos="0" name="service.family" value="MySQL"/>
+    <param pos="0" name="service.product" value="MariaDB Enterprise"/>
```

Comment:
In existing fingerprints this the `service.product` is `MariaDB` and the edition is `Enterprise Edition`.
`<param pos="0" name="service.edition" value="Enterprise Edition"/>`